src/IECoreScene/SceneCache.cpp

```diff
@@ -285,6 +245,8 @@ class SceneCache::Implementation : public RefCounted
 			}
 		}
 
+		IECore::CompoundDataBasePtr m_sets;
```

Comment:
As far as I can tell, this isn't used here in the base class, but is instead initialised in different ways in the derived classes. I'm guessing here, but in the reader I think it's used for converted tags? And then m_cachedSets is used for something else? If so, the naming is really unclear, as is having it in a class where it isn't used...